src/microbe_stage/Patch.cs

```diff
@@ -16,6 +16,9 @@
 [UseThriveSerializer]
 public class Patch
 {
+    [JsonIgnore]
+    private static readonly Compound Sunlight = SimulationParameters.Instance.GetCompound("sunlight");
```

Comment:
Also I'm kind of going to say that in this case this GetCompound call should be directly in the Patch constructors. This is because static compound value retrieves are a bit bad in terms of global game state and for example modding new compounds.